<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>添加任务</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="taskInput"/>
   </item>
   <item>
    <widget class="QListWidget" name="taskList"/>
   </item>
   <item>
    <widget class="QComboBox" name="priorityCombo">
     <item>
      <property name="text">
       <string>高</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>低</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
